include/stlcache/stlcache.hpp

```diff
@@ -32,6 +33,8 @@ using namespace std;
 #include <stlcache/policy_lfuagingstar.hpp>
 #include <stlcache/policy_adaptive.hpp>
 
+mutex mtx;
```

Comment:
Mutex need to be a part of stlcache class. What would happen, if someone will create two instances of stlcache sharing same mutex?